go/vt/servenv/http.go

```diff
@@ -46,6 +49,14 @@ func HTTPServe(l net.Listener) error {
 
 // HTTPRegisterProfile registers the default pprof HTTP endpoints with the internal servenv mux.
 func HTTPRegisterProfile() {
+	if !httpPprof {
+		return
+	}
+
+	if !pflag.Lookup("pprof-http").Changed {
+		log.Warning("Beginning in vitess version 20, pprof-http will default to `false`; to continue enabling pprof endpoints, please manually set this flag before upgrading.")
```

Comment:
Do we have a standard casing for `vitess`? Or `Vitess`? And should we use `Vitess 20`, `Vitess v20` as a shorter version instead of writing `version` out?

Reply:
I .... am not sure!

Looks like we definitely prefer `v20` to `version 20`:

```
go/vt/vtctl/reparentutil/emergency_reparenter.go:       // TODO(timvaillancourt): remove legacyERS* gauges in v19+.
go/vt/vttablet/onlineddl/executor.go:   // v18 fix. Remove in v19
go/vt/vttablet/onlineddl/executor.go:                   // This fix is created in v18 and can be removed in v19
go/vt/vttablet/tabletmanager/vreplication/flags.go:     // Deprecated and ignored in v19.
go/vt/vttablet/tabletmanager/vreplication/flags.go:     fs.MarkDeprecated("vreplication_tablet_type", "As of v19 this is ignored and will be removed in a future release.")
go/vt/vttablet/tabletserver/health_streamer.go: // tables and views in v19.
```

and it _seems_ like we mostly prefer capitalized Vitess but i'm not completely sure. let me make those changes!

Reply:
actually .. looking at the flag deprecation above, let's just drop "vitess" from the language entirely and avoid the issue :P